config/sidekiq.rb

```diff
@@ -0,0 +1,12 @@
+Sidekiq.configure_server do |config|   
+    config.redis = { url: 'redis://127.0.0.1:6379' } 
+  end
+  Sidekiq.configure_client do |config|   
+    config.redis = { url: 'redis://127.0.0.1:6379' } 
+  end
+  
+  schedule_file = 'config/schedule.yml'
+  if File.exist?(schedule_file) && Sidekiq.server?
+    Sidekiq::Cron::Job.load_from_hash YAML.load_file(schedule_file)
+  end
```

Comment:
This part should be inside `Sidekiq.configure_server` block - we want this to only load in sidekiq server(which is the thing we start with `bundle exec sidekiq`). And I believe it should be specified to be started only on startup - I can't link for specific part of code, but just above [Finding jobs](https://github.com/ondrejbartas/sidekiq-cron#finding-jobs) you will find example from repo how to load this to sidekiq-cron 🙂

Reply:
And could you fix the indentation in this file, please? 🙏

Reply:
My bad, saw this config and it worked so i didn't check the official documentation, should be more careful.